lib/chai/utils/inspect.js

```diff
@@ -31,15 +31,9 @@ function inspect(obj, showHidden, depth, colors) {
 }
 
 // Returns true if object is a DOM element.
-var isDOMElement = function (object) {
-  if (typeof HTMLElement === 'object') {
+function isDOMElement(object) {
+  if (typeof HTMLElement !== 'undefined') {
```

Comment:
What if `HTMLElement` is `undefined`?
The MDN docs aren't very clear about which browsers support this or not, there's a lot of `?` on the compatibility table.